src/@batch-flask/ui/form/form-field/form-field.html

```diff
@@ -19,9 +19,9 @@
         <ng-content select="[blFormFieldSuffix]"></ng-content>
     </div>
 </div>
-<div class="hints-container" *ngIf="hints.length > 0">
+<div class="hints-container" aria-live="polite">
```

Comment:
Does this result in a visual change now that hints-container is always visible?

Reply:
No, I checked. The `aria-live` specification is that the content _within_ such regions is announced when it changes, not when those regions are added to the DOM. So they need to exist in place, even as empty containers, for AT to properly announce content changes.